Adding guidance for conscious language replacements (#109)
redhat-documentation/supplementary-style-guide: supplementary_style_guide/style_guidelines/general-formatting.adoc

When the format _day Month year_ causes a presentation or clarity issue, use _Month day, year_ (for example, October 3, 2019) instead.

[[conscious-language]]
== Conscious language

> The Conscious Language Group is supporting the Red Hat commitment to remove problematic language from our code, documentation, websites, and open source projects that Red Hat is involved with.

Reviewer: @ndeevy s/is supporting/supports

s/that Red Hat is involved with/with which Red Hat is involved

> The Conscious Language Group is supporting the Red Hat commitment to remove problematic language from our code, documentation, websites, and open source projects that Red Hat is involved with.

Author: ```suggestion
The Conscious Language Group supports the Red Hat commitment to remove problematic language from our code, documentation, websites, and open source projects with which Red Hat is involved.
```

When the format _day Month year_ causes a presentation or clarity issue, use _Month day, year_ (for example, October 3, 2019) instead.

[[conscious-language]]
== Conscious language

The Conscious Language Group is supporting the Red Hat commitment to remove problematic language from our code, documentation, websites, and open source projects that Red Hat is involved with.
 

> .blacklist and whitelist 

Reviewer 2: I feel like this heading should be at a higher level than the examples below. Maybe consider making it a real level 3 heading (`=== blacklist and whitelist`) so that it can be seen as an umbrella for all the examples that fall below it?

When possible, rewrite documentation to avoid these terms.

> .Example (before)

Reviewer 2: I had a little bit of trouble following these at first glance. It's not initially clear which are grouped together. I'd suggest another way of grouping them somehow.

Could consider having one heading and then using icons to make it clear which is wrong or right. For example:

```
.Example: Removing 'whitelist'
image:images/no.png[no] The following steps demonstrate adding a new rule to whitelist a custom binary.

image:images/yes.png[yes] The following steps demonstrate adding a new rule to allow a custom binary.
```

![image](https://user-images.githubusercontent.com/7443887/130504561-f6439245-75a8-4131-a877-de398827c7e7.png)

Author: Great idea. I agree, that will look much more clear. TY

Heat blacklists any servers in the list from receiving updated heat deployments. After the stack operation completes, any blacklisted servers remain unchanged. You can also power off or stop the `os-collect-config` agents during the operation.

.Example (after)
Heat excludes any servers in the list from receiving updated heat deployments. After the stack operation completes, any excluded servers remain unchanged. You can also power off or stop the `os-collect-config` agents during the operation.

.Example (before)

> You can exclude overcloud nodes from receiving an updated deployment. This means that the blacklisted nodes are isolated from the effects of the stack operation. 

Author: ```suggestion
The following steps demonstrate adding a new rule to whitelist a custom binary.
```

.Example (after)

> You can exclude overcloud nodes from receiving an updated deployment by listing them in an environment file. The nodes you list are isolated from the effects of the stack operation. 

Author: ```suggestion
The following steps demonstrate adding a new rule to allow a custom binary.
```

When it is not possible to remove the terms blacklist and whitelist, replace them with one of the following alternatives:

* blocklist / allowlist: This combination is recommended by _The IBM Style Guide_. Use this combination unless your product area has another specific replacement that is agreed between engineering leadership and your documentation team.
* denylist / allowlist
* blocklist / passlist

> For more information about the Conscious Language Group, see https://github.com/conscious-lang/conscious-lang-docs.

Reviewer 2: I'd suggest moving this piece of info up with the initial sentence that references the Conscious Language Group
